routing/index_graph_starter.hpp

```diff
@@ -4,97 +4,99 @@
 #include "routing/joint.hpp"
 #include "routing/route_point.hpp"
 
+#include "std/limits.hpp"
 #include "std/utility.hpp"
+#include "std/vector.hpp"
 
 namespace routing
 {
-// The problem:
-// IndexGraph contains only road points connected in joints.
-// So it is possible IndexGraph doesn't contain start and finish.
-//
-// IndexGraphStarter adds fake start and finish joint ids for AStarAlgorithm.
-//
+class FakeVertex final
```

Comment:
This class is needed for IndexGraphStarter only (and for tests over IndexGraphStarter). Please, move it to the beginning of the public section of the IndexGraphStarter (but after using declarations).